libs/agent-sm/agent-policy/src/main/java/org/opensearch/security/package-info.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/**
+ * Classes for running code in a privileged context
+ */
+package org.opensearch.security;
```

Comment:
Why not put this in the same `org.opensearch.secure_sm` package as the rest of the code in this library?

Reply:
Not particular reason. My rationale was that it would move from `java.security.AccessController` -> `org.opensearch.security.AccessController`, but it also makes sense to keep it consistent with the rest of this lib. 

I refactored this to `org.opensearch.secure_sm.AccessController`